policy/modules/system/systemd.te

```diff
@@ -487,6 +490,7 @@ selinux_get_enforce_mode(systemd_logind_t)
 
 storage_getattr_removable_dev(systemd_logind_t)
 storage_getattr_scsi_generic_dev(systemd_logind_t)
+storage_raw_read_fixed_disk(systemd_logind_t)
```

Comment:
These rules should be conditional, especially because of this.

Reply:
I updated my commit in order to try introducing a tunable named `systemd_logind_get_bootloader`, but this fails to build:
```
/usr/bin/checkmodule -m tmp/systemd.tmp -o tmp/systemd.mod
policy/modules/system/systemd.te:566:
ERROR 'syntax error' at token 'typeattribute' on line 54942:
	typeattribute systemd_logind_t fixed_disk_raw_read;
#line 566
/usr/bin/checkmodule:  error(s) encountered while parsing configuration
/usr/bin/checkmodule:  loading policy configuration from tmp/systemd.tmp
```

As far as I understand, there is a `neverallow` rule which makes attribute `fixed_disk_raw_read` mandatory in order to read fixed disks (and this is nice), but associating types to attributes cannot be conditional/tunable. How would you like to handle this?